scrapy/extensions/postprocessing.py

```diff
@@ -11,6 +11,16 @@
 
 
 class GzipPlugin:
+    """
+    Compresses received data with with :py:mod:`gzip` module.
```

Comment:
The plugins should be about compressing in a specific file format.

```suggestion
    Compresses received data using `gzip <https://en.wikipedia.org/wiki/Gzip>`_.
```

The use of a Python module from the standard library is an implementation detail, and should not be covered in the documentation. Even if we link to the Python documentation about parameters, we could technically reimplement the plugin in the future without relying on the standard library module, while supporting the same option names and values (hence it being OK to link to that documentation for option reference).